### PR TITLE
Update postcss.config.js

### PR DESCRIPTION
**Pull request by paul-vd**
Return empty array if the condition is not met to avoid a null/undefined PostCSS plugin.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.6s | ⚠️ +103ms |
| nodeModulesSize | 53 MB | 53 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| index.html gzip | 918 B | 918 B | ✓ |
| link.html gzip | 926 B | 926 B | ✓ |
| withRouter.html gzip | 914 B | 914 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 11.2s | 11.6s | ⚠️ +360ms |
| nodeModulesSize | 53 MB | 53 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _error.js gzip | 293 kB | 293 kB | -457 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 957 B | 957 B | ✓ |
| index.js gzip | 294 kB | 294 kB | ⚠️ +286 B |
| link.js gzip | 302 kB | 301 kB | -119 B |
| routerDirect.js gzip | 300 kB | 300 kB | ⚠️ +14 B |
| withRouter.js gzip | 300 kB | 300 kB | -101 B |
| Overall change | 1.49 MB | 1.49 MB | -377 B |

</details>

</details>

File: examples/with-tailwindcss/postcss.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: [
     'tailwindcss',
-    process.env.NODE_ENV === 'production'
+      ...(process.env.NODE_ENV === 'production'
```
Comment (timneutkens) [inline examples/with-tailwindcss/postcss.config.js]: Seems that this will spread the options object of purgecss into the main array 🤔

Comment (PaulPCIO) [reply examples/with-tailwindcss/postcss.config.js]: Oh yea missed that, thanks 🤦‍♂

**Review by paul-vd — COMMENTED** [review]
Fixes made to the array spread

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 11.9s | 11.6s | -294ms |
| nodeModulesSize | 52.8 MB | 52.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.77 kB | 4.77 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| index.html gzip | 917 B | 917 B | ✓ |
| link.html gzip | 925 B | 925 B | ✓ |
| withRouter.html gzip | 915 B | 915 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.4s | -312ms |
| nodeModulesSize | 52.8 MB | 52.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.77 kB | 4.77 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _error.js gzip | 294 kB | 294 kB | -426 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 958 B | 958 B | ✓ |
| index.js gzip | 294 kB | 294 kB | -398 B |
| link.js gzip | 301 kB | 302 kB | ⚠️ +912 B |
| routerDirect.js gzip | 300 kB | 300 kB | ⚠️ +113 B |
| withRouter.js gzip | 300 kB | 300 kB | -77 B |
| Overall change | 1.49 MB | 1.49 MB | ⚠️ +124 B |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 11.4s | 11.4s | ⚠️ +65ms |
| nodeModulesSize | 52.8 MB | 52.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.77 kB | 4.77 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| index.html gzip | 917 B | 917 B | ✓ |
| link.html gzip | 925 B | 925 B | ✓ |
| withRouter.html gzip | 915 B | 915 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| buildDuration | 12.5s | 11.8s | -628ms |
| nodeModulesSize | 52.8 MB | 52.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.24 kB | 6.24 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..c6c1.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.2 kB | 56.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.77 kB | 4.77 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.71 kB | 6.71 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | PaulPCIO/next.js patch-4 | Change |
| - | - | - | - |
| _error.js gzip | 293 kB | 294 kB | ⚠️ +669 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 958 B | 958 B | ✓ |
| index.js gzip | 294 kB | 294 kB | -224 B |
| link.js gzip | 301 kB | 301 kB | ⚠️ +55 B |
| routerDirect.js gzip | 300 kB | 300 kB | ⚠️ +297 B |
| withRouter.js gzip | 300 kB | 300 kB | ⚠️ +696 B |
| Overall change | 1.49 MB | 1.49 MB | ⚠️ +1.49 kB |

</details>

</details>

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Looks like linting is failing

**Comment by timneutkens** [comment]
Thanks!

**Comment by PaulPCIO** [comment]
FYI Update with tailwind built-in purge https://github.com/tailwindcss/tailwindcss/pull/1639

**Pull request merged**